core/field_checkbox.js

```diff
@@ -77,14 +77,22 @@ Blockly.FieldCheckbox.CHECK_CHAR = '\u2713';
 Blockly.FieldCheckbox.prototype.CURSOR = 'default';
 
 /**
- * Install this checkbox on a block.
+ * Create the block UI for this checkbox.
+ * @deprecated May 2019 Use FieldCheckbox.initView.
  */
 Blockly.FieldCheckbox.prototype.init = function() {
-  if (this.fieldGroup_) {
-    // Checkbox has already been initialized once.
-    return;
-  }
+  // Logging a warning about calling a deprecated method here would be
```

Comment:
Can you remove this declaration of init? Since it inherits from Field it should just call the super method instead if it's not declared.

Ditto for everything else that inherits from Field that you're updating to just call the super.